interface/patient_file/encounter/trend_form.php

```diff
@@ -135,7 +135,7 @@ function show_graph(table_graph, name_graph, title_graph)
   // For LBF the <td> has an id of label_id_$fieldid
   $(".graph").on("click", function(e){ show_graph(<?php echo js_escape($formname); ?>, this.id.substring(9), $(this).text()) });
 <?php } else { ?>
-  $(".graph").on("click", function(e){ show_graph('form_vitals', this.id, '$(this).text()') });
+  $(".graph").on("click", function(e){ show_graph('form_vitals', this.id, $(this).text()) });
```

Comment:
Is this from a different changeset?

Reply:
This is odd, it appears to come from this commit:
```
commit 98e0421cd113393a84210ed0cbdcacf71171fdfe
Author: INFEG <66429354+infeg-khaleela@users.noreply.github.com>
Date:   Fri May 30 12:22:05 2025 +0530

    Vitals Graph Fix (#8393)

M       interface/patient_file/encounter/trend_form.php
```

But I have no idea how it snuck in there because I branched it from your PR kojiromike/coverage-annotations.  I used github's gh command to pull the pr and create a new branch which now has me worried as it shouldn't have brought that in.